src/MEDS_DEV/models/cehrbert/requirements.txt

```diff
@@ -0,0 +1,6 @@
+cehrbert==1.3.4
+meds_reader==0.1.9
+meds-evaluation==0.0.2
```

Comment:
Why do you need this dependency? MEDS-DEV runs evaluation for you on the prediction outputs. Do you want to do it internally as well for some reason?

Reply:
Ahh, I see, it is so you can check the schema. Makes sense. Nevermind!